src/extension/options-page/Welcome/index.tsx

```diff
@@ -148,6 +148,13 @@ export default withRouter(function _WelcomePortal(props: RouteComponentProps<{ i
     const provePost = useValueRef(ProvePostRef)
     const identifier = useValueRef(IdentifierRef)
 
+    const onFinish = WelcomeActions.onFinish.bind({ props })
+    const closeDialog = (e: React.MouseEvent) => {
```

Comment:
I don't think it's the right way to close a dialog. This dialog is by-design to be always open, if you want to close it, just change the router to `/`.

Reply:
And we have a close button, why detect target source?